tsconfig.json

```diff
@@ -1,13 +1,13 @@
 {
-  "include": ["./src/**/*.ts", "./src/**/*.tsx"],
+  "include": ["./src/**/*.ts", "./src/**/*.tsx", "./tests/**/*.ts"],
   "exclude": ["node_modules", "dist"],
   "compilerOptions": {
     "outDir": "./dist/esm",
-    "rootDir": "./src",
+    "rootDir": "./",
```

Comment:
not sure, but potentially this way the tests get included in your `dist` folder and therefor can get published